ros_tcp_endpoint/service.py

```diff
@@ -15,6 +15,7 @@
 import rclpy
 import re
 import std_srvs
+import std_srvs.srv
 
```

Comment:
Would be best to import what is needed specifically 

```suggestion
from std_srvs.srv import Trigger
```
Would need to change the check accordingly. I think it would improve readability as well